.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -27,6 +27,17 @@ else
     echo "Running tests: Go files changed"
 fi
 
+function add_comment {
```

Comment:
This file has a lot of comments - maybe it would be nice either in this PR or a follow-up to switch over to this function for all of them? Maybe taking the comment content as an arg instead of hardcoding it?

Reply:
I thought this seemed familiar 😂 it looks like I did something similar here: https://github.com/GoogleCloudPlatform/magic-modules/pull/5049/commits/4dcefb374ac8d8395840f6be2801332a6ce8ecc2 - feel free to take that if you want!

Reply:
sorry, missed your second comment at first, but i updated it so i think it's pretty close to what you had :) thanks!